src/main/java/fr/xephi/authme/output/LogFilterService.java

```diff
@@ -0,0 +1,43 @@
+package fr.xephi.authme.output;
+
+import fr.xephi.authme.command.CommandMapper;
+import fr.xephi.authme.command.FoundCommandResult;
+
+import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Service class for the log filters.
+ */
+public class LogFilterService {
+
+    private static final String ISSUED_COMMAND_PREFIX_TEXT = "issued server command: /";
+
+    @Inject
+    private CommandMapper commandMapper;
+
+    /**
+     * Validate a message and return whether the message contains a sensitive AuthMe command.
+     *
+     * @param message The message to verify
+     *
+     * @return True if it is a sensitive AuthMe command, false otherwise
+     */
+    public boolean isSensitiveAuthMeCommand(String message) {
+        if (message == null || !message.contains(ISSUED_COMMAND_PREFIX_TEXT)) {
+            return false;
+        }
+        String rawCommand = message.substring(message.indexOf("/"));
+        List<String> components = Arrays.asList(rawCommand.split(" "));
+        FoundCommandResult command = commandMapper.mapPartsToCommand(null, components);
+        switch (command.getResultStatus()) {
```

Comment:
This is really cool! Do you think we should somehow cache invocations of the commands? I'm a little worried about performance since this will be performed every time someone issues a command, and I'm not quite sure how efficient the commandMapper is. 
Failing that, I just saw that the LogFilterHelper has a test (`LogFilterHelperTest`) that ensures that all of those previously hard-coded labels are present. The alternative would be to keep the list and to update the test to use the sensitive flag.

Reply:
1) yeah, performance can be a problem hmm... we should test it.
2) I'm not sure about the test :/ What would you do?

Reply:
hmm, yeah, which portion of logic do you want to move?